src/main/resources/assets/iris/lang/lol_us.json

```diff
@@ -1,33 +1,33 @@
-  "iris.keybind.toggleShaders": "Toggwe Shadews",
-  "iris.keybinds": "Iwis",
-  "iris.shaders.reloaded.failure": "Faiwed to wewoad shadews! Reason: %s",
-  "iris.shaders.toggled.failure": "Faiwed to toggwe shadews! Reason: %s",
-  "iris.sodium.failure.title": "Iwis faiwed to woad!",
-  "iris.sodium.failure.reason": "You have Sodium in youw mods fowdew, howevew Iwis incwudes it. Pwease cwose da game, and wemove Sodium from youw mods fowdew.",
+  "iris.shaders.reloaded": "Shaedews wewoded!",
+  "iris.shaders.toggled": "Turnd Blek cat un grownd to %s!",
+  "iris.keybind.reload": "Rewoaded Blek cat un gownd",
+  "iris.keybind.shaderPackSelection": "Blek Ket On Grownd Bramd Selectur scren",
+  "iris.keybind.toggleShaders": "Toggwe Blek Cat Un Ground",
+  "iris.keybinds": "Iris",
+  "iris.shaders.reloaded.failure": "COWLDNT REWOAD BLECK KET ON GROWND! Reson: %s",
+  "iris.shaders.toggled.failure": "COWLDNT FIND BLECK KET TO PUT UN GROWND! Reson: %s",
+  "iris.sodium.failure.title": "Iris cowld nut be able to woad!",
+  "iris.sodium.failure.reason": "You hab Sodium in kiten's mods fowder, however Iris includes it. Pleas cwose da game, and rembve Sodium from kitten's mods fowder.",
 
-  "options.iris.apply": "Appwy",
-  "options.iris.refresh": "Refwesh",
-  "options.iris.openShaderPackFolder": "Open Shadew Pack Fowdew...",
-  "options.iris.refreshShaderPacks": "Refwesh Shadew Packs",
-  "options.iris.shaderPackSelection": "Shadew Packs...",
-  "options.iris.shaderPackSelection.title": "Shadew Packs",
-  "options.iris.shaderPackSelection.addedPack": "Added Shadew Pack \"%s\"",
-  "options.iris.shaderPackSelection.addedPacks": "Added %s Shadew Packs",
-  "options.iris.shaderPackSelection.failedAdd": "Fiwes wewe not vawid Shadew Packs",
-  "options.iris.shaderPackSelection.failedAddSingle": "\"%s\" ish not a vawid Shadew Pack",
+  "options.iris.apply": "DUN YET?!",
+  "options.iris.refresh": "Reflesh",
+  "options.iris.openShaderPackFolder": "Open Blek Ket Veriants Fowder...",
+  "options.iris.refreshShaderPacks": "Reflesh Blek Kets",
+  "options.iris.shaderPackSelection": "Blek Kets Dat Go On Grownd...",
+  "options.iris.shaderPackSelection.title": "Blek Kets",
+  "options.iris.shaderPackSelection.addedPack": "Aded Blek Ket \"%s\"",
+  "options.iris.shaderPackSelection.addedPacks": "Aded %s Blek Kets",
+  "options.iris.shaderPackSelection.failedAdd": "Deez kitens were not real blek kets",
+  "options.iris.shaderPackSelection.failedAddSingle": "\"%s\" is not real blek ket",
   "options.iris.shaderPackSelection.copyError": "Couwd not add Shadew Pack \"%s\"",
-  "options.iris.shaderPackSelection.copyErrorAlreadyExists": "\"%s\" ish awwready in youw Shadew Packs fowdew!",
-  "options.iris.shaders.disabled": "Shadews: Disabwed",
-  "options.iris.shaders.enabled": "Shadews: Enabwed",
-  "options.iris.shadowDistance": "Mwax Swadow Dwistance",
+  "options.iris.shaderPackSelection.copyErrorAlreadyExists": "\"%s\" is alredy ur blek ket!",
```

Comment:
"U alredy got \"%s\" in yr shadur packz foldur!!"

Reply:
nice